erica/infrastructure/sqlalchemy/alembic/README.md

```diff
@@ -0,0 +1,13 @@
+Generic single-database configuration.
+
+### create new revision
+
+```bash
+alembic revision -m"create erica request table"
```

Comment:
We could add a task to the `task.py`. This would make it easy to use that feature and go into the correct directory and we can use that task for the Github actions


We could reference the task in the normal READE.md as part of instructions to set the project up.

Reply:
At first I was not sure if I need to run alembic in this directory or if I can run it anywhe

Reply:
ok.
The alembic.ini lives in the root dir.